components/playground/instance/pd.go

```diff
@@ -61,12 +62,16 @@ func (inst *PDInstance) Start(ctx context.Context, version repository.Version) e
 	args := []string{
 		"tiup", compVersion("pd", version),
 		"--name=" + uid,
-		fmt.Sprintf("--data-dir=%s", filepath.Join(inst.Dir, "data")),
-		fmt.Sprintf("--peer-urls=http://%s:%d", inst.Host, inst.Port),
-		fmt.Sprintf("--advertise-peer-urls=http://%s:%d", inst.Host, inst.Port),
-		fmt.Sprintf("--client-urls=http://%s:%d", inst.Host, inst.StatusPort),
-		fmt.Sprintf("--advertise-client-urls=http://%s:%d", inst.Host, inst.StatusPort),
-		fmt.Sprintf("--log-file=%s", filepath.Join(inst.Dir, "pd.log")),
+	}
+	if inst.ConfigPath != "" {
```

Comment:
We can remove other flags even if we specify a configuration file. We should keep the original flags and append `--config` if the config flag exists.

Reply:
Thank you. Fixed.